apps/passport-client/components/screens/LoginScreens/AlreadyRegisteredScreen.tsx

```diff
@@ -133,16 +132,15 @@ export function AlreadyRegisteredScreen() {
         appConfig.zupassServer,
         encryptionKey
       );
-      setIsLoggingIn(false);
 
       if (!storageResult.success) {
+        setIsLoggingIn(false);
         return setError(
           "Password incorrect. Double-check your password. " +
             "If you've lost access, you can reset your account below."
         );
       }
 
-      setIsLoadingUser(true);
       dispatch({
```

Comment:
oh one last thing, - let's `await` this dispatch and wrap it in a `try` block, then `setIsLoggingIn(false)` + `setError(e.message)` within the catch